lib/readline.js

```diff
@@ -120,8 +119,8 @@ function Interface(input, output, completer, terminal) {
   this.input = input;
   this.historySize = historySize;
   this.removeHistoryDuplicates = !!removeHistoryDuplicates;
-  this.crlfDelay = Math.max(kMincrlfDelay,
-                            Math.min(kMaxcrlfDelay, crlfDelay >>> 0));
+  this.crlfDelay = crlfDelay === Infinity ?
```

Comment:
I fixed this with 
`this.crlfDelay = crlfDelay ? Math.max(kMincrlfDelay, crlfDelay) : kMincrlfDelay;`
to make sure `crlfDelay` is not `undefined`, and add one unit test to check `this.crlfDelay` can be float number.
`crlfDelay` is a pure JavaScript variable, won't pass to C++.